react/src/lib/components/DeckGLMap/layers/map/mapLayer.stories.tsx

```diff
@@ -196,6 +196,39 @@ MapLayer2d.parameters = {
     },
 };
 
+export const MapLayer2dDarkMode: ComponentStory<typeof DeckGLMap> = (args) => {
+    return <DeckGLMap {...args} />;
+};
+
+MapLayer2dDarkMode.args = {
+    id: "map",
+    layers: [
+        axes_hugin,
+        { ...meshMapLayerFloat32, material: false },
+        north_arrow_layer,
+    ],
+    bounds: [432150, 6475800, 439400, 6481500] as NumberQuad,
+    views: {
+        layout: [1, 1],
+        viewports: [
+            {
+                id: "view_1",
+                show3D: false,
+            },
+        ],
+    },
+};
+
+MapLayer2dDarkMode.parameters = {
+    docs: {
+        ...defaultParameters.docs,
+        description: {
+            story: "Example using png as mesh and properties data.",
```

Comment:
It would be good if you give bit clear description in line 226 like in bracket dark mode or whatever you prefer, just to know that this is done for dark mode. Just an suggestion, you can also keep as it is, no problem :)